src/platform/forms-system/src/js/utilities/validations/index.js

```diff
@@ -32,13 +34,7 @@ export function isValidSSN(value) {
 }
 
 export function isValidYear(value) {
-  return (
-    Number(value) >= 1900 &&
-    Number(value) <=
-      moment()
-        .add(100, 'year')
-        .year()
-  );
+  return Number(value) >= minYear && Number(value) <= maxYear;
```

Comment:
I did a double take when I saw this function and the same `isValidYear` in the _other_ validations file :see_no_evil: